presto-docs/src/main/sphinx/system-requirements/software-requirements.rst

```diff
@@ -33,3 +33,18 @@ For more on SSH configuration, see :ref:`ssh-configuration-label`.
 **Other Configuration**
 
 * Sudo privileges on both the node running ``presto-admin`` and the nodes where Presto will be installed are required for a non-root presto-admin user.
+* Number of open file descriptors and user processes for ``presto`` user should be set to the following values in ``/etc/security/limits.conf`` otherwise you may run into this error "java.lang.OutOfMemoryError: unable to create new native thread"
```

Comment:
s/Number/The number
s/`presto`/the `presto`/

I'd say, put "otherwise..." below the limits in the jmx catalog